Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -4,6 +4,7 @@ class << self
       OPTIMIZATION_FLAGS_LINUX = {
         core2: "-march=core2",
         core: "-march=prescott",
+        arm: "-march=armv7-a",
```

Comment:
For a Raspberry Pi version 1, I believe it should be `-march=armv6`
See https://raspberrypi.stackexchange.com/questions/2046/which-cpu-flags-are-suitable-for-gcc-on-raspberry-pi